buildkite/helm/buildkite-exporter/templates/_healthchecks.common.tpl

```diff
@@ -0,0 +1 @@
+../../../../helm/common/templates/_healthchecks.tpl
```

Comment:
May make sense to define within a common Mina library chart (vs. using symlinks)